rqt_py_common/src/rqt_py_common/topic_completer.py

```diff
@@ -46,6 +46,24 @@ def __init__(self, parent=None):
     def splitPath(self, path):
         # to handle array subscriptions, e.g. /topic/field[1]/subfield[2]
         # we need to separate array subscriptions by an additional /
+
+        topic_name = path
+
+        if self.topic_list:
+            # Remove backslash at the end of the topic name
+            if (topic_name[-1] == '/'):
+                topic_name = topic_name[:-1]
+
+            for topic, type in self.topic_list:
```

Comment:
Do you need the `type`?

Reply:
https://github.com/ros-visualization/rqt/pull/299/commits/26b0fe84d35368e035b2a004b8628d8c3824ba54